Wikipedia/Code/ViewWithFindInPageKeyboardBarInputAccessoryView.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import "FindInPageKeyboardBar.h"
+
+@interface ViewWithFindInPageKeyboardBarInputAccessoryView : UIView
```

Comment:
According to the referenced docs, you need a subclass of `UIResponder`. That means you can use `UIViewController` as well (it is part of the responder chain as well)

There is an example of accomplishing this task without the extra view here:
https://robots.thoughtbot.com/input-accessorizing-uiviewcontroller